src/views/UserSettings/Inner.jsx

```diff
@@ -94,48 +85,29 @@ const Inner = memo(
-                                            render={renderProps => (
-                                                <Button
-                                                    type="link"
-                                                    className="user-settings-form__btn--padding-x-0"
-                                                    danger={
-                                                        userInfo.facebookStatus
-                                                    }
-                                                    onClick={
-                                                        userInfo.facebookStatus
-                                                            ? handleUnlinkFacebook
-                                                            : renderProps.onClick
-                                                    }
-                                                    onChange={
-                                                        userInfo.facebookStatus
-                                                            ? null
-                                                            : renderProps.callback
-                                                    }
-                                                    disabled={
-                                                        renderProps.isProcessing ||
-                                                        facebookProcessing
-                                                    }
-                                                >
-                                                    {renderProps.isProcessing ||
-                                                    facebookProcessing
-                                                        ? 'Đang xử lý...'
-                                                        : userInfo.facebookStatus
-                                                        ? 'Hủy liên kết Facebook'
-                                                        : 'Liên kết Facebook'}
-                                                </Button>
-                                            )}
-                                        />
+                                        <Button
+                                            type="link"
+                                            className="user-settings-form__btn--padding-x-0"
+                                            danger={userInfo.facebookStatus}
+                                            onClick={
+                                                userInfo.facebookStatus
+                                                    ? handleUnlinkFacebook
+                                                    : handleLinkFacebook
+                                            }
+                                            disabled={facebookProcessing}
+                                        >
+                                            {facebookProcessing
```

Comment:
k dùng toán tử 3 ngôi liên tiếp như này

Reply:
Chỗ này mình chưa biết sửa sao hết. Đặt một biến displayText rồi dùng useState ư?

Reply:
dùng condition riêng cho từng text
{facebookProcessing && 'Đang xử lý...'}
...
tương tự cho 2 cái dưới